tests/change-key-removes-user.sh

```diff
@@ -50,3 +55,4 @@ echo "$username:SHA256:badfingerprint" > "$userfile"
 
 # check the authorized_keys is empty
 test "$(cat "$authorized_keys")" = ""
+echo "test: $?"
```

Comment:
Is this left in by mistake? Shouldn't ever by non-zero, should it?

Reply:
Yep, left over debugging, will remove